configtx/config_test.go

```diff
@@ -19,6 +19,29 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+func TestNewConfigTx(t *testing.T) {
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+
+	channel, _, err := baseApplicationChannelGroup(t)
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	original := &cb.Config{
+		ChannelGroup: channel,
+	}
+
+	c := New(original)
+	gt.Expect(proto.Equal(c.OriginalConfig(), original)).To(BeTrue())
+	gt.Expect(proto.Equal(c.UpdatedConfig(), original)).To(BeTrue())
+
+	err = c.Application().AddCapability("fake-capability")
+	gt.Expect(err).NotTo(HaveOccurred())
+
+	gt.Expect(proto.Equal(c.OriginalConfig(), original)).To(BeTrue())
+	gt.Expect(proto.Equal(c.UpdatedConfig(), original)).To(BeFalse())
```

Comment:
Would probably like a true assertion against an expected updated instead but I'm fine merging as is 😛

Reply:
We have that everywhere else in the tests (just directly using c.updated instead of c.UpdatedConfig()). :) 

I figured this test should simply ensure 1) the configs are proto.Equal after creating the ConfigTx and 2) that they differ after updating something.